app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from __future__ import annotations
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks_to_buy += 1
+    if masks_to_buy > 0:
```

Comment:
According to the checklist, you should avoid comparing a variable to 0 if there is no need. Instead of `if masks_to_buy > 0:`, you can simply write `if masks_to_buy:`.